### PR TITLE
Simple maintenance improvements

### PR DESCRIPTION
**Pull request by a1346054**

### CODE REVIEW

File: src/greetermenubar.c [hunk 1]
```diff
@@ -46,15 +46,15 @@ sort_minimal_size(gconstpointer a, gconstpointer b, GtkRequestedSize* sizes)
...
-    GList 			 *item;
-    GList 			 *shell_children;
-    GList 			 *expand_nums = NULL;
-    guint 			  visible_count = 0;
-	guint 			  expand_count = 0;
+    GtkPackDirection  pack_direction;
+    GList             *item;
+    GList             *shell_children;
+    GList             *expand_nums = NULL;
+    guint             visible_count = 0;
```
Comment (ochosi) [inline src/greetermenubar.c:53]:
```suggestion
    guint              visible_count = 0;
```

File: src/greetermenubar.c [hunk 2]
```diff
@@ -46,15 +46,15 @@ sort_minimal_size(gconstpointer a, gconstpointer b, GtkRequestedSize* sizes)
...
+    guint             expand_count = 0;
```
Comment (ochosi) [inline src/greetermenubar.c:54]:
```suggestion
    guint              expand_count = 0;
```

File: src/greetermenubar.c [hunk 3]
```diff
@@ -46,15 +46,15 @@ sort_minimal_size(gconstpointer a, gconstpointer b, GtkRequestedSize* sizes)
 static void
 greeter_menu_bar_size_allocate(GtkWidget* widget, GtkAllocation* allocation)
 {
-	GtkPackDirection  pack_direction;
-    GList 			 *item;
-    GList 			 *shell_children;
-    GList 			 *expand_nums = NULL;
-    guint 			  visible_count = 0;
-	guint 			  expand_count = 0;
+    GtkPackDirection  pack_direction;
```
Comment (ochosi) [inline src/greetermenubar.c:49]:
```suggestion
    GtkPackDirection   pack_direction;
```

**Review by ochosi — CHANGES_REQUESTED** [review]
Thanks for the cleanups and typo fixes. At least in GitHub's UI it looks as though the variable names without leading * are missing a space (the first variable character should always be aligned).
Please verify/change. Thanks!

Otherwise I think this is ok to merge as is @bluesabre

**Comment by a1346054** [comment]
Fixed up that commit with your suggestions, well spotted, thanks.

**Review by ochosi — APPROVED** [review]

**Pull request merged**